README.md

```diff
@@ -48,8 +49,34 @@ When using the `-D`/`--dist_est` ARCS option to estimate gap sizes, the user is
 
 An example bash script on how to run the ARCS+LINKS pipeline can be found at: Examples/pipeline_example.sh
 
+### Running ARCS in default mode
+
+Default mode used read alignments to contigs.
+
+For running the pipeline default mode, Makefile should target `arcs` with `./arcs-make arcs`. For more info check `./arcs-make help`.
+
+To run `./arcs` executable in default mode, run `./arcs` without additional flag. For argument requiriments check `./arcs --help`.
+
+### Running ARCS in '--arks' mode
+
+ARKS is another scaffolding tool developed by our group designed based on kmerization paradigm. ARKS is integrated into ARCS(v1.x.x) for better usability.
+
+For running the pipeline with ARKS, Makefile should target `arks` with `./arcs-make arks`. For more info check `./arcs-make help`.
+
+To run `./arcs` executable with ARKS, run `./arcs --arks`. For more info check `./arcs --help`.
+
+### Demo
+
 you can test your installation by following instructions at: Examples/arcs_test-demo/README.txt
+for ARKS: Examples/arks_test-demo/README.txt
+
```

Comment:
You can remove the rest of the demo text below here -- I think it is redundant with my suggestion above.